.github/workflows/build.yml

```diff
@@ -93,8 +93,11 @@ jobs:
       - name: Build bundles
         run: npm run build-bundles
       # install OS dependencies required by browsers on the GitHub runner
+      # to be sure that the browser is correctly installed: https://github.com/microsoft/playwright/issues/5801
       - name: Install the dependencies for all browsers
-        run: npx playwright install-deps
+        run: |
+          npx playwright install
```

Comment:
⚠️ if we download and install the browser here, we should skip browser installation during the `npm ci` by setting the `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD` environment variable to `1`